src/nft/components/profile/list/NFTListRow.tsx

```diff
@@ -20,7 +19,7 @@ const NFTListRowWrapper = styled(Row)`
   border-radius: 8px;
 
   &:hover {
-    background: ${({ theme }) => opacify(24, theme.backgroundOutline)};
+    background: ${({ theme }) => theme.backgroundOutline};
```

Comment:
note: the definition of `theme.backgroundOutline` is already `opacify(24, theme.backgroundOutline)`